tests/backends/test_awkward.py

```diff
@@ -26,97 +27,120 @@
 ) < packaging.version.Version("2.2.3")
 
 
-def test_dimension_conversion():
+def _assert(bool1, bool2, backend):
+    # for 'typetracer' backend we pass through (no-op)
+    # for 'cpu' we want to make sure the bool evaluates to True
+    if backend == "cpu":
+        if isinstance(bool1, ak.Array):
+            bool1 = bool1.tolist()
+        if isinstance(bool2, ak.Array):
+            bool2 = bool2.tolist()
+        else:
```

Comment:
This just falls through and does nothing if bool2 is a ak.Array?

Reply:
it calls `.tolist()` if it's an `ak.Array`